src/Table/Column/FilterModel/TypeString.php

```diff
@@ -13,7 +13,7 @@ protected function init(): void
     {
         parent::init();
 
-        $this->op->values = ['is' => 'Is', 'contains' => 'Contains', 'start' => 'Start with', 'end' => 'End with'];
+        $this->op->values = ['is' => 'Is', 'is not' => 'Is Not', 'contains' => 'Contains', 'start' => 'Start with', 'end' => 'End with'];
```

Comment:
and not contains, not starts... Not sure if this is wanted...

Reply:
At least simple "Is Not" definitely is good to have and also - it was already implemented there, but wasn't added in op->values array :)